packages/esm-patient-registration-app/src/patient-registration/field/cause-of-death/cause-of-death.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useConfig } from '@openmrs/esm-framework';
+import { type RegistrationConfig } from '../../../config-schema';
+import styles from '../field.scss';
+import { useConceptAnswers } from '../field.resource';
+import classNames from 'classnames';
+import { Field } from 'formik';
+import { Layer, Select, SelectItem, SkeletonPlaceholder } from '@carbon/react';
+
+export const CauseOfDeath: React.FC = () => {
+  const { t } = useTranslation();
+  const {
+    fieldConfigurations: {
+      causeOfDeath: { conceptUuid, validation },
+    },
+  } = useConfig<RegistrationConfig>();
+
+  const { data: conceptAnswers, isLoading: isLoadingConceptAnswers } = useConceptAnswers(conceptUuid);
+
+  const answers = useMemo(
+    () => (isLoadingConceptAnswers ? [] : conceptAnswers.map((answer) => ({ ...answer, label: answer.display }))),
+    [conceptAnswers, isLoadingConceptAnswers],
+  );
+
+  if (isLoadingConceptAnswers) {
+    return <SkeletonPlaceholder className={styles.skeletonPlaceholder} />;
+  }
+
+  return (
+    <div className={classNames(styles.customField, styles.halfWidthInDesktopView)}>
+      <h4 className={styles.productiveHeading02Light}>{t('causeOfDeathInputLabel', 'Cause of Death')}</h4>
+      <Field name="deathCause">
+        {({ field, form: { touched, errors }, meta }) => {
+          return (
+            <Layer>
+              <Select
+                id="deathCause"
+                name="deathCause"
+                labelText={t('causeOfDeathInputLabel', 'Cause of Death')}
+                required={validation?.required}
+                invalid={errors.deathCause && touched.deathCause}
+                {...field}>
```

Comment:
```suggestion
                {...field}
                id="deathCause"
                name="deathCause"
                labelText={t('causeOfDeathInputLabel', 'Cause of Death')}
                required={validation?.required}
                invalid={errors.deathCause && touched.deathCause}
                >
```

`{...field}` must be destructured at the top, otherwise, the field props will override your props.